src/engraving/dom/segment.cpp

```diff
@@ -2251,6 +2251,23 @@ EngravingItem* Segment::prevElement(staff_idx_t activeStaff)
     }
 }
 
+EngravingItem* Segment::firstElement(track_idx_t startTrack, track_idx_t endTrack) const
```

Comment:
looks like you are only using this method in Segment::firstElement(staff_idx_t staffIdx), so probably you should leave only one